modules/gdscript/gd_parser.cpp

```diff
@@ -2014,7 +2023,7 @@ void GDParser::_parse_block(BlockNode *p_block,bool p_static) {
 			} break;
 			default: {
 
-				Node *expression = _parse_and_reduce_expression(p_block,p_static,false,true);
+				Node *expression = _parse_and_reduce_expression(p_block,p_static,true,true);
```

Comment:
I'm not really sure why this was left to false (it is `p_reduce_const`), would be nice if @reduz confirms :smiley: